Android/app/build.gradle

```diff
@@ -5,11 +5,11 @@ android {
     aaptOptions {
         noCompress "webp"
     }
-    compileSdkVersion 29
+    compileSdkVersion 30
     defaultConfig {
-        applicationId "com.example.samplestickerapp"
+        applicationId "com.simonxu.samplestickerapp"
```

Comment:
Let revert to `com.example.samplestickerapp`

Reply:
yes, good catch.